docs/source/best-practices/checkpointing.rst

```diff
@@ -74,8 +86,53 @@ To submit this array job to the scheduler, use the following command::
 
    sbatch submit_mdrun_array.bash
 
-Python TensorFlow checkpointing example
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Checkpointing using DMTCP
```

Comment:
For consistency, this should either be DMTCP checkpointing example, or the previous heading should be Checkpointing using GROMACS.